.github/workflows/ci.yaml

```diff
@@ -7,11 +7,38 @@ on:
     branches: [main]
 
 jobs:
+  ruff-versions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          sudo apt-get install -y curl jq ripgrep
```

Comment:
curl and jq are preinstalled: https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md

Reply:
Oh cool!